src/views/TradingCompetition/components/YourScore/CardUserInfo.tsx

```diff
@@ -0,0 +1,212 @@
+import React, { ReactText } from 'react'
+import {
+  Text,
+  Heading,
+  Flex,
+  Skeleton,
+  MedalBronzeIcon,
+  MedalGoldIcon,
+  MedalPurpleIcon,
+  MedalSilverIcon,
+  MedalTealIcon,
+} from '@pancakeswap-libs/uikit'
+import styled from 'styled-components'
+import useI18n from 'hooks/useI18n'
+import { YourScoreProps } from '../../types'
+import UserRankBox from './UserRankBox'
+import NextRankBox from './NextRankBox'
+import { localiseTradingVolume } from '../../helpers'
+
+const TeamRankTextWrapper = styled(Flex)`
+  svg {
+    width: 24px;
+  }
+`
+
+const RanksWrapper = styled(Flex)`
+  width: 100%;
+  margin-top: 24px;
+  flex-direction: column;
+
+  ${({ theme }) => theme.mediaQueries.sm} {
+    flex-direction: row;
+  }
+`
+
+const CardUserInfo: React.FC<YourScoreProps> = ({ hasRegistered, account, profile, userLeaderboardInformation }) => {
+  const TranslateString = useI18n()
+  // eslint-disable-next-line camelcase
```

Comment:
`next_rank` is a value returned by the API

Reply:
In that case
```
{ next_rank: nextRank }
```

Reply:
Going to sort in follow-up PR, might be good to transform the data when its fetched from the API? Otherwise we have `next_rank` being passed as props, until its used by a component where it becomes `nextRank` - or do you think that's OK?